mlflow/exceptions.py

```diff
@@ -62,6 +62,10 @@ def serialize_as_json(self):
     def get_http_status_code(self):
         return ERROR_CODE_TO_HTTP_STATUS.get(self.error_code, 500)
 
+    @classmethod
+    def invalid_parameter_value(cls, message, **kwargs):
```

Comment:
```suggestion
    def invalid_parameter_value(cls, message, **kwargs):
        """
        Constructs an `MlflowException` object with `error_code=INVALID_PARAMETER_VALUE`.
        """
```

Can we add a docstring?